src/app/view/Auth/ErrorTokenModal.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import EventEmitter from 'events'
+
+import { act, fireEvent, render, screen } from '@testing-library/react-native'
+import React from 'react'
+import { Text } from 'react-native'
+
+import { ErrorTokenModal } from '/app/view/Auth/ErrorTokenModal'
+import { useRevokedStatus } from '/app/view/Auth/useRevokedStatus'
+import { translation } from '/locales'
+import { AuthService } from '/app/domain/authentication/services/AuthService'
+
+// Mock functions
+const eventEmitter = new EventEmitter()
+let userRevokedStatus = false
+type MockService = typeof AuthService
+
+const mockService: MockService = {
+  isUserRevoked: jest.fn(() => userRevokedStatus),
+  setUserRevoked: jest.fn((status: boolean) => {
+    userRevokedStatus = status
+    status
+      ? eventEmitter.emit('userRevoked', status)
+      : eventEmitter.emit('userLoggedOut', status)
+  }),
+  startListening: jest.fn(),
+  emitter: eventEmitter
+}
+
+const mockIsUserRevoked = mockService.isUserRevoked as jest.Mock
+
+// Stub Component using the hook
+const ComponentUsingHook = ({
+  authService
+}: {
+  authService: MockService
+}): JSX.Element => {
+  const { userRevoked, userConfirmation } = useRevokedStatus(authService)
+
+  return userRevoked ? (
+    <ErrorTokenModal onClose={userConfirmation} />
+  ) : (
+    <Text>All fine</Text>
+  )
+}
+
+describe('ErrorTokenModal and useRevokedStatus', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('renders ErrorTokenModal when revoked status is true', () => {
```

Comment:
Those tests are not testing `ErrorTokenModal` but `ComponentUsingHook`. As the conditional rendering is in this mocked component.

Reply:
This test is removed as I don't see any benefit in testing a simple modal now that the hook is removed